frontend/src/screens/learner/studies.ts

```diff
@@ -52,13 +51,10 @@ export const useParticipantStudies = () => {
         allHighlightedStudies.length == FEATURED_COUNT ? [] : randomlyHighlighted
     )
 
-    const nonHighlightedStudies = studies.filter(s => !highlightedStudies.includes(s))
```

Comment:
With study paths, highlighted studies can be shown in both places now, so we dont need to filter them out from the rest of the studies below the highlighted section